app/views/cases/_event.html.erb

```
@@ -0,0 +1,9 @@
+<div class="card event-card">
+  <div class="card-header d-inline-flex justify-content-between align-items-center">
+    <span><%= date.to_formatted_s(:long) %> - <b><%= name %></b></span>
+    <%= icon(defined?(type) ? type : 'info-circle') %>
```

Do we need a default for `type`? AFAICS we're not using this at the moment, and it would be useful to get an error rather than a default if we ever don't pass a `type` to this template, which will force us to decide what icon we want to show for that event.

---

Thanks for making the change - I tend to favour "fail fast (in development) so we're forced to decide how we want to handle the new case" but agree it's very situation-dependent :slightly_smiling_face: